data/eFLOWER_Dun_2022/raw/eFLOWER_Dun_2022 extra R code.R

```diff
@@ -0,0 +1,31 @@
+read_csv("data/eFLOWER_Dun_2022/raw/qry_Lily_Data_AusTraits_2023-05-21.csv") %>%
+  subset(!is.na(Character_state)) %>% 
+  mutate(trait_value = Character_state) -> character_values
+
+read_csv("data/eFLOWER_Dun_2022/raw/qry_Lily_Data_AusTraits_2023-05-21.csv") %>%
+  subset(!is.na(Value)) %>% 
+  subset(is.na(Max)) %>%
```

Comment:
I wrote this script for the first eFLOWER input and had to remember by thought process!! It is because there are some instances (very few) with identical mean and max values and then I just wanted to retain one and chose to call it a max

Reply:
Oh I see! I just searched for those and I can see there are 16 instances where that happens (identical mean, min and max values) but there's one instance where there's a mean, min and max that are all different. It's for Atherosperma moschatum and the character '331. Number of androecium structural whorls (C1)' (NDat 45221).

I also noticed for Idiospermum australiense and character '401. Number of structural carpels (C1)' (NDat 2684) the minimum is 2 while the mean is 1. From the notes column maybe it should be min 0, mean 1 and max 2? With such a large dataset there's bound to be tiny mistakes but I thought I may as well identify them when I notice them.